platform/android/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 7.2.0 - February 27, 2019
+
+### Bugs
+- fix surface creation after app backgrounding for surfaceview[#13976](https://github.com/mapbox/mapbox-gl-native/pull/13976)
+
+### Features
+- make ReLinker default library loader[#13974](https://github.com/mapbox/mapbox-gl-native/pull/13974)
```

Comment:
This is not a part of the `kombucha` release.

Reply:
Removed